boot/boot.go

```diff
@@ -435,17 +435,43 @@ func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *Boota
 		return fmt.Errorf("cannot write modeenv: %v", err)
 	}
 
-	// update recovery grub's grubenv to indicate that we transition
-	// to run mode now
+	// get the ubuntu-boot bootloader
 	opts := &bootloader.Options{
+		// TODO:UC20: we use "recovery: true" here because on
+		// the partition the file layout of ubuntu-boot looks
+		// the same as ubuntu-seed.
+		// TODO:UC20: need a better name than recovery
+		Recovery: true,
```

Comment:
I thought a bit more about this, my preference for the moment is to have a 2nd flag:

BootNotMounted

or something like that. Internally it has the same behavior as Recovery, but it refers to a run env, not a recovery env.

I think this all stay a bit in flux until we start supporting a different enough 2nd bootloader.